app/components/CalendarRefresh.js

```diff
@@ -0,0 +1,18 @@
+const React = require('react');
+const { defineSvg } = require('../defineSvg');
+
+module.exports = defineSvg('CalendarRefresh', {
+  className: 'tilt-icon-calendar-refresh',
+
+  render() {
```

Comment:
'concise methods' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).